wdl/gwas/saige_sub.multiproc.wdl

```diff
@@ -0,0 +1,141 @@
+task test {
+
+	File nullfile
+	File varianceratiofile = sub(nullfile, ".rda", ".varianceRatio.txt")
+    String outfileprefix = basename(nullfile, ".rda") + "-"
+	Array[File] bgenfiles
+	File samplefile
+	Int minmac
+	String docker
+    String loco
+
+    command {
+
+        python3 <<EOF
+        import os
+        import shlex
+        import subprocess
+        processes = set()
+        cmd_prefix = 'step2_SPAtests.R \
+            --minMAC=${minmac} \
+            --sampleFile=${samplefile} \
+            --GMMATmodelFile=${nullfile} \
+            --varianceRatioFile=${varianceratiofile} \
+            --numLinesOutput=1000 \
+            --IsOutputAFinCaseCtrl=TRUE \
+            --LOCO=${loco} '
+        for file in '${sep=" " bgenfiles}'.split(' '):
+            cmd = cmd_prefix + '--bgenFile=' + file
+            cmd = cmd + ' --SAIGEOutputFile=${outfileprefix}' + os.path.basename(file) + '.SAIGE.txt'
+            logfile = open('SAIGE_log_${outfileprefix}' + os.path.basename(file) + '.txt', 'w')
+            processes.add(subprocess.Popen(shlex.split(cmd), stdout=logfile))
+        for p in processes:
+            if p.poll() is None:
+                p.wait()
+        EOF
+    }
+
+	output {
+        Array[File] out = glob("*.SAIGE.txt")
+        Array[File] logs = glob("SAIGE_log_*.txt")
+    }
+
+    runtime {
+        docker: "${docker}"
+        cpu: length(bgenfiles)
+        memory: (3.75 * length(bgenfiles)) + " GB"
+        disks: "local-disk " + (length(bgenfiles) * ceil(size(bgenfiles[0], "G")) + 1) + " HDD"
```

Comment:
Add sort by size suggestion to markdown. Also there could be option to give bgen chunk size as parameter and len(bgens) that amount would be reserved.... that would be just for ease of configuration...anyhoo at least document